spyder/plugins/appearance/widgets.py

```diff
@@ -87,12 +90,14 @@ def validate_colors(self):
                 invalid_colors[key] = items[0].text()
 
         if invalid_colors:
-            message = _("The following properties have invalid colors:\n\n")
+            message = _(
+                "The following properties have invalid colors:<br><br>"
+                )
```

Comment:
```suggestion
            )
```

Align with the variable name above.